go/oasis-node/cmd/debug/fixgenesis/fixgenesis.go

```diff
@@ -247,7 +246,27 @@ func convertRuntime(rt *oldRuntime) *registry.Runtime {
 			CheckpointInterval:      rt.Storage.CheckpointInterval,
 			CheckpointNumKept:       rt.Storage.CheckpointNumKept,
 			CheckpointChunkSize:     rt.Storage.CheckpointChunkSize,
-			MinPoolSize:             rt.Storage.GroupSize,
+		},
+		Constraints: map[scheduler.CommitteeKind]map[scheduler.Role]registry.SchedulingConstraints{
```

Comment:
how do we work this file? is there a process where we 'reset' it after we use it to migrate to another version? or are we maintaining it to be a tool for translating from some specific old genesis version?

Reply:
This is specifically from 20.12 to master.

Reply:
ok